python/pyspark/ml/tuning.py

```diff
@@ -143,6 +145,8 @@ def getEvaluator(self):
 
 class CrossValidator(Estimator, ValidatorParams):
     """
+    .. note:: Experimental
```

Comment:
Those are kmeans docs? This is for `CrossValidator`. There are so many doc PRs currently it's confusing :)

I see the Scaladoc is not great:

```
/**
 * :: Experimental ::
 * K-fold cross validation.
 */
```
So this already matches. It would be nice to have better docs for both Python and Scala, but let's not worry about it now.